vtex/checkout-ui-tests
# CHK-1061: Remove broken tests and update documentation

#### What is the purpose of this pull request?

1. Remove all remaining broken tests (that is, tests that never pass)
2. Update documentation mostly by removing unused or outdated parts

#### What problem is this solving?

Nowadays when running the entire test suite, it's hard to know which tests failed due to a new feature/bugfix or if it's a test that is broken for a really long time. By fixing all tests, that problem would go away, but not all tests were easy to fix, so we opted to just discard those tests:

- \[Shipping\] Delivery + Scheduled Delivery and Delivery - Credit card (vtexgame1, clean, geo, invoice, nolean)
- [Shipping Preview] Delivery + Scheduled Delivery + Pickup (nolean)
- [Shipping Preview] Delivery + Scheduled Delivery + Scheduled Pickup (nolean)
- [Shipping Preview] Delivery Only + Delivery Pickup (nolean)

We could spend some effort to fix them, but there are other known issues with more priority. You can read more details about what is happening in each test [here](https://www.notion.so/vtexhandbook/Ressurrei-o-de-testes-E2E-e1b3fcea081342e2bcd1defedd0dddab).

Besides that, I updated the documentation where I think it was necessary (like removing `SCENARIOS.md`, which was very outdated).

#### How should this be manually tested?

I don't think there is need to be manually test, but you can check [the Cypress Dashboard run for this branch](https://dashboard.cypress.io/projects/kobqo4/runs/5514/) :satisfaction:
